src/pytest/test_conditional_updates.py

```diff
@@ -30,11 +30,11 @@ def testConditionalUpdateOnNoneExistingNumericField(env):
     env.assertEqual(env.cmd('ft.add', 'idx', 'doc1', 1.0,
                               'REPLACE', 'PARTIAL',
                               'IF', '@id1 != @id2',
-                              'fields', 'id1', '3', 'id2', '4'), 'NOADD')
+                              'fields', 'id1', '3', 'id2', '4'), 'OK')
```

Comment:
Same, you break behavior, how does it behave or 1.4?

Reply:
@MeirShpilraien @mnunberg @gkorland 
In 1.4 it behaves the same as before but this seems like wrong behavior.

Reply:
I agree but changing now is a breaking change